app/controllers/oauth2_controller.rb

```diff
@@ -7,6 +7,7 @@ class Oauth2Controller < ApplicationController
 
   force_ssl if: -> { RailsApiAuth.force_ssl }
 
+  # rubocop:disable all
```

Comment:
Only the method length check should be disable and only for this specific method (enable it again after the method definition).